src/main/resources/default/taglib/w/multiLanguageTextField.html.pasta

```diff
@@ -0,0 +1,97 @@
+<i:arg name="defaultLanguage" type="String" default="@NLS.getCurrentLang()"/>
+<i:arg name="validLanguages" type="List" default="@value.getValidLanguages()"/>
+<i:arg name="manageLanguages" type="boolean" default="false"/>
+
+<i:pragma name="description" value="Renders a multi language text input field within a Wondergem template"/>
+
+<i:local name="functionName" value="@('openMultiLanguageModal' + fieldName)"/>
+<i:local name="fieldId" value="@('mls-field-' + id)"/>
+<i:local name="modalId" value="@('mls-modal-' + id)"/>
+
+<div id="@fieldId" class="col-xs-@smallSpan col-md-@span form-group @UserContext.get().signalFieldError(fieldName)">
+    <i:if test="isFilled(label) || forceLabel">
+        <label>
+        <span class="@if (required) { input-required } @if (adminOnly) { admin-link }">
+            @label
+        </span>
+        </label>
+    </i:if>
+    <div class="input-group" id="@('insert-point-' + fieldName)" data-toggle="modal" data-target="#___modalId">
+        <input id="@fieldId"
+               name="@fieldId"
+               type="@type"
+               value="@value.fetchTextOrFallback(defaultLanguage)"
+               class="form-control input-block-level mls-input pointer"
+               readonly
+               @if (isFilled(placeholder)) { placeholder="@placeholder" }
+               @if (autofocus) { autofocus }
+               @if (isFilled(tabIndex)) { tabindex="@tabIndex" }/>
+        <span class="input-group-addon pointer"><span class="fa fa-globe"></span></span>
+    </div>
+    <div id="@('hidden-inputs-' + fieldName)" class="mls-hidden-inputs">
+    </div>
+
+    <i:if test="isFilled(help)">
+        <span class="help-block"><i:raw>@help</i:raw></span>
+    </i:if>
+    <i:if test="UserContext.get().hasError(fieldName)">
+        <span class="error-block help-block">@UserContext.get().getFieldErrorMessage(fieldName)</span>
+    </i:if>
+</div>
+
+<w:modal name="@modalId" title="@label" scrollable="true">
+    <w:modalBody>
+        <div class="mls-modal-inputs">
+
+        </div>
+    </w:modalBody>
+    <w:modalFooter>
+        <div class="mls-add-language-button btn-group dropup pull-left hidden">
+            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false">
+                <i class="fa fa-plus"></i> @i18n("MultiLanguageEditor.addLanguage")
+            </button>
+            <ul class="dropdown-menu"></ul>
+        </div>
+
+        <a class="btn btn-primary ok-btn">@i18n("NLS.close")</a>
+    </w:modalFooter>
+</w:modal>
+
+<script>
+    $(document).ready(function () {
+        let validLanguages = {};
+        <i:for type="List" var="lang" items="validLanguages">
+            validLanguages['@lang'] = '@i18n("Language." + lang)';
+        </i:for>
+
+        let field = new MultiLanguageField({
+            fieldName: '___fieldName',
+            hasFallback: ___value.isWithFallback(),
```

Comment:
```suggestion
            hasFallback: ___value.hasFallback(),
```

Reply:
hasFallback wasn't the right method here as it's used internally and also checks if a fallback object is contained inside the mls data. Therefore a new getter was defined called isWithFallback - so no error here.

Reply:
OK 👍 
I wasn't aware of that change